--- criu-2.4/criu/Makefile~	2016-08-21 10:55:35.000000000 +0300
+++ criu-2.4/criu/Makefile	2016-08-21 10:55:36.728867281 +0300
@@ -48,7 +48,7 @@
         ifneq ($(shell sh -c                                                            \
                         'TMP="$(OUTPUT)$(TMPOUT).$$$$";                                 \
                         echo "int main(int argc, char *argv[]) { return 0; }" |         \
-                        "$(CC)" -x c - $(LIBS) -o "$$TMP" > /dev/null 2>&1 && echo y;   \
+                        $(CC) -x c - $(LIBS) -o "$$TMP" > /dev/null 2>&1 && echo y;   \
                         rm -f "$$TMP"'),y)
                 $(warning "Couldn't find some of the required libraries")
                 $(warning "Make sure the following packages are installed")
