--- criu-1.7/Makefile.orig	2015-09-12 20:13:41.777088038 +0200
+++ criu-1.7/Makefile	2015-09-12 20:54:17.423652491 +0200
@@ -351,7 +351,7 @@
 
 install-crit: crit
 	$(E) "  INSTALL crit"
-	$(Q) python scripts/crit-setup.py install --root=$(DESTDIR) --prefix=$(PREFIX)
+	$(Q) python scripts/crit-setup.py install --root=$(DESTDIR) --prefix=$(PREFIX) --optimize=2
 
 .PHONY: install install-man install-crit install-criu
 
